examples/cross_todo/nim_commandline/nimtodo.nim

```diff
@@ -42,30 +42,27 @@ type
     cmdGenerate       # Add random rows to the database, for testing.
     cmdList           # User wants to list contents.
 
-  TParamConfig = object
+  ParamConfig = object
     # Structure containing the parsed options from the commandline.
-    command: TCommand         # Store the type of operation
+    command: Command         # Store the type of operation
     addPriority: int          # Only valid with cmdAdd, stores priority.
     addText: seq[string]      # Only valid with cmdAdd, stores todo text.
     todoId: int64             # The todo id for operations like check or delete.
     listParams: TPagedParams  # Uses the backend structure directly for params.
 
-
-proc initDefaults(params: var TParamConfig) =
+proc initDefaults(params: var ParamConfig) =
   ## Initialises defaults value in the structure.
   ##
   ## Most importantly we want to have an empty list for addText.
   params.listParams.initDefaults
   params.addText = @[]
 
-
 proc abort(message: string, value: int) =
   # Simple wrapper to abort also displaying the help to the user.
   stdout.write(USAGE)
   quit(message, value)
 
-
-template parseTodoIdAndSetCommand(newCommand: TCommand): stmt =
+template parseTodoIdAndSetCommand(newCommand: Command): typed =
```

Comment:
Same mistake?

Reply:
Is there a compelling reason not to make a 1-for-1 swap of _typed_ for _stmt_ here or should it be replaced it with _untyped_?

Reply:
I'm fairly sure ``stmt`` == ``untyped``.

Reply:
This is the lead I followed:

https://nim-lang.org/docs/manual.html#templates-typed-vs-untyped-parameters

"Note: For historical reasons *stmt* is an alias for *typed* and *expr* an alias for *untyped*, but new code should use the newer, clearer names."

So I did a 1-to-1 replacement except where *{.immediate.}* called for *untyped*

Reply:
huh, I guess I am horribly wrong.

Reply:
I'd use an _untyped_ in my code but significant revision of the examples beyond trimming a few legacy stylistic and syntactic is beyond the scope of this particular PR. 

I'm considering submitting PRs featuring some curated Rosetta Code examples for this directory but any proposal I make for that is still in incubation.

Reply:
Currently return types should always be ``untyped`` though.

Reply:
Changed to untyped